runtime/rococo/constants/src/lib.rs

```diff
@@ -97,15 +99,16 @@ mod tests {
 		currency::{CENTS, DOLLARS, MILLICENTS},
 		fee::WeightToFee,
 	};
+	use crate::weights::ExtrinsicBaseWeight;
 	use frame_support::weights::{DispatchClass, WeightToFeePolynomial};
-	use runtime_common::BlockWeights;
+	use runtime_common::MAXIMUM_BLOCK_WEIGHT;
 
 	#[test]
-	// This function tests that the fee for `MaximumBlockWeight` of weight is correct
+	// This function tests that the fee for `MAXIMUM_BLOCK_WEIGHT` of weight is correct
 	fn full_block_fee_is_correct() {
 		// A full block should cost 16 DOLLARS
-		println!("Base: {}", BlockWeights::get().get(DispatchClass::Normal).base_extrinsic);
-		let x = WeightToFee::calc(&BlockWeights::get().max_block);
+		println!("Base: {}", ExtrinsicBaseWeight::get());
+		let x = WeightToFee::calc(&MAXIMUM_BLOCK_WEIGHT);
```

Comment:
Test here looks weird so I changed it to look like in the other runtimes; [in polkadot](https://github.com/paritytech/polkadot/blob/bb02373ef87d5d1efb84a9f22c1e32a71e000fc7/runtime/polkadot/constants/src/lib.rs#L109), [kusama](https://github.com/paritytech/polkadot/blob/bb02373ef87d5d1efb84a9f22c1e32a71e000fc7/runtime/kusama/constants/src/lib.rs#L106) and [westend](https://github.com/paritytech/polkadot/blob/bb02373ef87d5d1efb84a9f22c1e32a71e000fc7/runtime/westend/constants/src/lib.rs#L108) it is all done this way.  
Could also be macro generated :shrug: